snuba/migrations/operations.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, storage_set: StorageSetKey, table_name: str) -> None:
         self.table_name = table_name
 
     def format_sql(self) -> str:
-        return f"DROP TABLE IF EXISTS {self.table_name};"
+        return f"DROP TABLE IF EXISTS {self.table_name} SYNC;"
```

Comment:
same here about supported by older clickhouse versions. If there are a bunch of places like this, we may want to have like a setting that enables the new features or something

Reply:
removed